.travis.yml

```diff
@@ -15,7 +15,6 @@ env:
     - RAILS_ENV=test
     - CC_TEST_REPORTER_ID=f39e060a8b1a558ebd8aff75d5b9760bf1ae98f3f85d628ae28814f3c66438cd
     - DATABASE_URL=postgres://postgres@localhost/
-    - COVERAGE=true
```

Comment:
For additional color, this is being removed bc [we are now testing coverage by default](https://github.com/thepracticaldev/dev.to/pull/4666/files )